CRM/Financeextras/Hook/BuildForm/RefundButton.php

```diff
@@ -0,0 +1,49 @@
+<?php
+
+use CRM_Financeextras_ExtensionUtil as E;
+
+/**
+ * Implement Refund Button if supported Refund by Payment Processor
+ */
+class CRM_Financeextras_Hook_BuildForm_RefundButton {
```

Comment:
I think the class name is a bit vague.  Class name should be related to the form. Maybe `CRM_Financeextras_Hook_BuildForm__AdditionalPaymentButton`,  as we could re-use this class to inject another button like   Refund by direct debit etc.  

Please ensure you're following coding standards.
```
https://docs.civicrm.org/dev/en/latest/standards/#civicrm-vs-drupal
https://www.drupal.org/node/608152#naming
```

So in Drupal, naming conventions for class name should be as define as  
`Classes and interfaces should have names that stand alone to tell what they do without having to refer to the namespace, read well, and are as short as possible without losing functionality information or leading to ambiguity. Notes:`